doc/start/hardware-recommendations.rst

```diff
@@ -75,30 +75,30 @@ Memory
 ======
 
 Bluestore uses its own memory to cache data rather than relying on the
-operating system page cache.  In bluestore you can adjust the amount of memory
-the OSD attempts to consume with the :confval:`osd_memory_target` configuration
-option.
+operating system's page cache. In bluestore you can adjust the amount of memory
+that the OSD attempts to consume by changing the :confval:`osd_memory_target`
+configuration option.
 
 - Setting the :confval:`osd_memory_target` below 2GB is typically not
-  recommended (it may fail to keep the memory that low and may also cause
-  extremely slow performance.
+  recommended (Ceph may fail to keep the memory consumption under 2GB and 
+  this may cause extremely slow performance).
 
 - Setting the memory target between 2GB and 4GB typically works but may result
   in degraded performance: metadata may be read from disk during IO unless the
   active data set is relatively small.
 
-- 4GB is the current default :confval:`osd_memory_target` size.  This default
+- 4GB is the current default :confval:`osd_memory_target` size. This default
   was chosen for typical use cases, and is intended to balance memory
   requirements and OSD performance for typical use cases.
```

Comment:
`typical use cases` is in this sentence twice.  Suggest truncating after `performance` while you're in this file.

Reply:
Good point.